<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MailEditorWindow</class>
 <widget class="QMainWindow" name="MailEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTextEdit" name="messageEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mailFieldsToolBar">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>72</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Mail fields</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Standard</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>21</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="adjustToolbar">
   <property name="windowTitle">
    <string>Adjust</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLeft"/>
   <addaction name="actionCenter"/>
   <addaction name="actionRight"/>
   <addaction name="actionJustify"/>
  </widget>
  <widget class="QToolBar" name="mailToolBar">
   <property name="windowTitle">
    <string>Mail attachements</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAttachement"/>
   <addaction name="actionMoney"/>
  </widget>
  <widget class="QToolBar" name="mailFieldSelectorToolBar">
   <property name="windowTitle">
    <string>Recipient fields</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="formatToolBar">
   <property name="windowTitle">
    <string>Format</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
  </widget>
  <widget class="QToolBar" name="fileAttachementToolBar">
   <property name="windowTitle">
    <string>File attachement</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="moneyAttachementToolBar">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>34</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Money attachement</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Keyhotee.qrc">
     <normaloff>:/images/win/filesave.png</normaloff>:/images/win/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="iconText">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save document</string>
   </property>
   <property name="statusTip">
    <string>Save currently edited message into Drafts folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>:images/win/editcopy.png</normaloff>:images/win/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy current selection to clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-cut" resource="../Keyhotee.qrc">
     <normaloff>:/images/win/editcut.png</normaloff>:/images/win/editcut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-paste" resource="../Keyhotee.qrc">
     <normaloff>:/images/win/editpaste.png</normaloff>:/images/win/editpaste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSend">
   <property name="icon">
    <iconset theme="mail-send" resource="../Keyhotee.qrc">
     <normaloff>:/images/128x128/contact_info_send_mail.png</normaloff>:/images/128x128/contact_info_send_mail.png</iconset>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="toolTip">
    <string>Allows to send given mail document</string>
   </property>
   <property name="statusTip">
    <string>Allows to send currently edited message</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Keyhotee.qrc">
     <normaloff>:/images/win/textleft.png</normaloff>:/images/win/textleft.png</iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Keyhotee.qrc">
     <normaloff>:/images/win/textcenter.png</normaloff>:/images/win/textcenter.png</iconset>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Keyhotee.qrc">
     <normaloff>:/images/win/textright.png</normaloff>:/images/win/textright.png</iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionJustify">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Keyhotee.qrc">
     <normaloff>:/images/win/textjustify.png</normaloff>:/images/win/textjustify.png</iconset>
   </property>
   <property name="text">
    <string>Justify</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Keyhotee.qrc">
     <normaloff>:/images/win/textbold.png</normaloff>:/images/win/textbold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="toolTip">
    <string>Changes selected text to heavier font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Keyhotee.qrc">
     <normaloff>:/images/win/textitalic.png</normaloff>:/images/win/textitalic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Changes currently selected text to &lt;span style=&quot; font-style:italic;&quot;&gt;italic&lt;/span&gt; font&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Keyhotee.qrc">
     <normaloff>:/images/win/textunder.png</normaloff>:/images/win/textunder.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="toolTip">
    <string>Draw line below selected text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionMailFields">
   <property name="icon">
    <iconset resource="../Keyhotee.qrc">
     <normaloff>:/images/gear.png</normaloff>:/images/gear.png</iconset>
   </property>
   <property name="text">
    <string>Mail fields</string>
   </property>
   <property name="toolTip">
    <string>Allows to select additional mail fields (Bcc: Cc: From:)</string>
   </property>
  </action>
  <action name="actionFrom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>From:</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CC:</string>
   </property>
  </action>
  <action name="actionBCC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BCC:</string>
   </property>
  </action>
  <action name="actionAttachement">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Keyhotee.qrc">
     <normaloff>:/images/paperclip-icon.png</normaloff>:/images/paperclip-icon.png</iconset>
   </property>
   <property name="text">
    <string>Attachement</string>
   </property>
   <property name="toolTip">
    <string>Allows to manage mail attachement</string>
   </property>
  </action>
  <action name="actionMoney">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Keyhotee.qrc">
     <normaloff>:/images/money-in-envelope.png</normaloff>:/images/money-in-envelope.png</iconset>
   </property>
   <property name="text">
    <string>Money</string>
   </property>
   <property name="toolTip">
    <string>Allows to attach money to this mail</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-undo" resource="../Keyhotee.qrc">
     <normaloff>:/images/128x128/undo.png</normaloff>:/images/128x128/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-redo" resource="../Keyhotee.qrc">
     <normaloff>:/images/128x128/redo.png</normaloff>:/images/128x128/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Keyhotee.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>messageEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>actionCopy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>messageEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>actionCut</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopy</sender>
   <signal>triggered()</signal>
   <receiver>messageEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCut</sender>
   <signal>triggered()</signal>
   <receiver>messageEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPaste</sender>
   <signal>triggered()</signal>
   <receiver>messageEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUndo</sender>
   <signal>triggered()</signal>
   <receiver>messageEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>messageEdit</sender>
   <signal>redoAvailable(bool)</signal>
   <receiver>actionRedo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>messageEdit</sender>
   <signal>undoAvailable(bool)</signal>
   <receiver>actionUndo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>messageEdit</sender>
   <signal>cursorPositionChanged()</signal>
   <receiver>MailEditorWindow</receiver>
   <slot>onCursorPositionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRedo</sender>
   <signal>triggered()</signal>
   <receiver>messageEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelect_all</sender>
   <signal>triggered()</signal>
   <receiver>messageEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBCC</sender>
   <signal>toggled(bool)</signal>
   <receiver>MailEditorWindow</receiver>
   <slot>onBccTriggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCC</sender>
   <signal>toggled(bool)</signal>
   <receiver>MailEditorWindow</receiver>
   <slot>onCcTriggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFrom</sender>
   <signal>toggled(bool)</signal>
   <receiver>MailEditorWindow</receiver>
   <slot>onFromTriggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MailEditorWindow</receiver>
   <slot>onSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adjustToolbar</sender>
   <signal>actionTriggered(QAction*)</signal>
   <receiver>MailEditorWindow</receiver>
   <slot>onTextAlignTriggerred(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>-5</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBold</sender>
   <signal>triggered(bool)</signal>
   <receiver>MailEditorWindow</receiver>
   <slot>onTextBoldTriggerred(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionItalic</sender>
   <signal>triggered(bool)</signal>
   <receiver>MailEditorWindow</receiver>
   <slot>onTextItalicTriggerred(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUnderline</sender>
   <signal>triggered(bool)</signal>
   <receiver>MailEditorWindow</receiver>
   <slot>onTextUnderlineTriggerred(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAttachement</sender>
   <signal>triggered(bool)</signal>
   <receiver>MailEditorWindow</receiver>
   <slot>onFileAttachementTriggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMoney</sender>
   <signal>triggered(bool)</signal>
   <receiver>MailEditorWindow</receiver>
   <slot>onMoneyAttachementTriggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCursorPositionChanged()</slot>
  <slot>onBccTriggered(bool)</slot>
  <slot>onCcTriggered(bool)</slot>
  <slot>onFromTriggered(bool)</slot>
  <slot>onSave()</slot>
  <slot>onTextAlignTriggerred(QAction*)</slot>
  <slot>onTextBoldTriggerred(bool)</slot>
  <slot>onTextUnderlineTriggerred(bool)</slot>
  <slot>onTextItalicTriggerred(bool)</slot>
  <slot>onFileAttachementTriggered(bool)</slot>
  <slot>onMoneyAttachementTriggered(bool)</slot>
 </slots>
</ui>
